piano.yml

```diff
@@ -307,7 +307,7 @@ Layer:
             AND type IN ('state', 'city', 'town')
             AND geometry && !bbox!
           ORDER BY
-            prio
+            prio, population DESC
```

Comment:
![capture d ecran 2018-02-23 a 09 50 27](https://user-images.githubusercontent.com/178744/36585664-15fa7bc0-187f-11e8-9c61-8929a0529bb4.png)

Valencia in Spain is much more bigger than Silla